packages/embark-ui/src/components/EnsLookup.js

```diff
@@ -46,7 +46,7 @@ class EnsLookup extends Component {
         </CardHeader>
         <CardBody>
           <FormGroup>
-            <Input placeholder="Enter an address" onChange={e => this.handleChange(e)}/>
+            <Input placeholder="Enter an address (0x..)" onChange={e => this.handleChange(e)}/>
```

Comment:
```suggestion
            <Input placeholder="Enter an address (0x...)" onChange={e => this.handleChange(e)}/>
```
For consistency's sake, an ellipsis should always have three dots.